### ISSUE TITLE
Rasa Language Embeddings

### ISSUE THREAD

**Issue by nelealbers** [PerfectFit-project/virtual-coach-issues#150]
Explore the addition of pretrained Rasa language embeddings to improve generalization of training examples etc (see here: https://rasa.com/docs/rasa/tuning-your-model/).

**Comment by svenvanderburg** [comment]
I can recommend this series of videos explaining DIET architecture: https://www.youtube.com/watch?v=vWStcJDuOUk

This is what I got from it:
* Indeed using pretrained embeddings (spacy or BERT) will very likely improve our chatbot, since we have little training data and our data does not reflect real conversations.
* I think we need a bit more training/test data to do proper benchmarking
* We could take out the CRF part that deals with entities, since we don't need entity recognition. At the very least we can take out the `EntitySynonymMapper` to speed up the pipeline.
* We can disable masking, since our training data does not reflect real conversations and it is small.

**Comment by nelealbers** [comment]
I compared a bit with and without the small dutch language model.

What I get is that without the language model, we get better generalization to spelling errors and grammar ones (e.g., ik heb net gespoort), while with the language model we get better generalization to other words (e.g., "ik voel me ook super" instead of "ik voel me ook goed" as in the nlu-file works).
